app/views/generic_files/upload/_tos_checkbox.html.erb

```diff
@@ -0,0 +1,3 @@
+<label style="display:none;">
+  <%= check_box_tag 'terms_of_service', 1, 1, {data: {activate: 'activate-submit'}} %><strong> I have read and do agree to <%= link_to t('sufia.deposit_agreement'), '/agreement/', target: "_blank" %>.</strong>
```

Comment:
We could probably also overwrite this method and just return true.

https://github.com/projecthydra/sufia/blob/a064489d3cf4d228abf978d7126644e2c653aa5e/app/controllers/concerns/sufia/files_controller_behavior.rb#L246